packages/turbo-workspace-convert/src/cli.ts

```diff
@@ -0,0 +1,187 @@
+    --summary       Show a summary of the workspace at <dir>, including the detected package
+                    manager, all workspaces, and their locations
+    --help, -h      Show this help message
+    --version, -v   Show the version of this script
+`;
+
+run().catch((err: Error) => {
+  if (err?.cause !== "expected") {
+    console.log("Unexpected error - aborting");
+    console.error(err.message || err);
+    process.exit(1);
+  }
+  console.log();
+  console.error(err.message || err);
+  process.exit(0);
+});
+
+async function run() {
+  let { input, flags, showHelp, showVersion } = meow(help, {
+    booleanDefault: undefined,
+    flags: {
+      npm: { type: "boolean", default: false },
+      pnpm: { type: "boolean", default: false },
+      yarn: { type: "boolean", default: false },
+      install: { type: "boolean" },
+      dry: { type: "boolean", default: false },
+      summary: { type: "boolean", default: false },
+      help: { type: "boolean", default: false, alias: "h" },
+      version: { type: "boolean", default: false, alias: "v" },
+    },
+  });
+
+  if (flags.help) showHelp();
+  if (flags.version) showVersion();
+  const logger = new Logger({ dry: flags.dry, interactive: true });
+
+  logger.hero();
+  await new Promise((resolve) => setTimeout(resolve, 500));
+  logger.header("Welcome, let's convert your project.");
+  logger.blankLine();
+  // Figure out the app directory
+  let projectDir = path.resolve(
+    process.cwd(),
+    input.length > 0
+      ? input[0]
+      : (
+          await inquirer.prompt<{ dir: string }>([
+            {
+              type: "input",
+              name: "dir",
+              message: "Where is the root of your monorepo?",
+              default: ".",
+            },
+          ])
+        ).dir
+  );
+
+  let shouldInstall = flags.install;
+  if (shouldInstall === undefined && !flags.dry) {
+    const installResponse = await inquirer.prompt<{ install: string }>([
+      {
+        type: "list",
+        name: "install",
+        message: "Should we automatically install after conversion?",
+        default: "yes",
+        choices: ["yes", "no"],
+      },
+    ]);
+    shouldInstall = installResponse.install === "yes";
+  }
+
+  // if it's absolute, don't worry about it
+  const workspaceRoot = path.isAbsolute(projectDir)
+    ? projectDir
+    : path.relative(process.cwd(), projectDir);
+  const project = await getWorkspaceDetails({ workspaceRoot });
+
+  // log workspace summary
+  if (flags.summary) {
+    logger.workspaceSummary({ project, workspaceRoot });
+  }
+
+  if (flags[project.packageManager]) {
+    throw new Error("You are already using this package manager", {
+      cause: "expected",
+    });
+  }
+
+  let answers: Answers;
+  if (flags.npm) {
+    answers = { packageManager: "npm" };
+  } else if (flags.pnpm) {
+    answers = { packageManager: "pnpm" };
+  } else if (flags.yarn) {
+    answers = { packageManager: "yarn" };
+  } else {
```

Comment:
This is why they shouldn't be individual flags.

`convert --pnpm --npm` will _very_ confusingly return `npm`.